paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -775,12 +783,37 @@ if(WITH_MKLDNN)
     ${GRU_DATA_PATH})
   # run post-training quantization lexical analysis test
   inference_analysis_api_lexical_int8_test_run(
-    test_analyzer_lexical_gru_int8 ${LEXICAL_TEST_APP} ${GRU_MODEL_PATH}
-    ${GRU_DATA_PATH} false)
+    test_analyzer_lexical_gru_int8
+    ${LEXICAL_TEST_APP}
+    ${GRU_MODEL_PATH}
+    ${GRU_DATA_PATH}
+    true
+    false
+    false)
```

Comment:
I'm not a fan of naked bool flags, could you place it in the documenting variable?
You could also use a comment for it.